tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java

```diff
@@ -231,6 +231,9 @@ public long estimateMaxSeriesMemSize() {
 
   @Override
   public long getCurrentChunkSize() {
+    if (this.getCurrentDataSize() == 0) {
```

Comment:
Is this check necessary?

Reply:
You may save the calculation result and use it in line238 to avoid duplicated calculation.

And if all chunks in a chunk group are empty, this chunk group should not be flushed into the file, which will generate an empty ChunkGroupMetadata and waste some space.

Reply:
`this.getCurrentDataSize()` actually is `pageBuffer.size()` while `pageBuffer` call `outputstream.count`.
Therefore, I remove getCurrentDataSize() and use pageBuffer.size() instead.

@qiaojialin , yes I think so. 
No matter whether you check the serialized size in flushAll... method, this method is needed and should check the data size here.